app/assets/stylesheets/components/main.scss

```diff
@@ -30,6 +30,10 @@ main {
   ul {
     list-style-type: disc;
     padding-left: 20px;
+
+    li {
+      margin-bottom: 10px;
```

Comment:
I tend to use `em` measurements for this sort of thing, so that the spacing is relative to the size of the text it's applied to. 

Maybe replace with with `0.5em` ?